### PR TITLE
Automatically push to travis on commits to release branch

### PR DESCRIPTION
**Pull request by aggFTW**
Developer will have to bump versions/requirements for every package manually based on rules outlined at http://semver.org/

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,13 +1,44 @@
...
+    condition: $TRAVIS_PYTHON_VERSION = "2.7"
+- provider: script
+  script: /bin/sh deploy_test.sh
+  on:
+    branch: release
+    tags: false
+    condition: $TRAVIS_PYTHON_VERSION = "2.7"
+env:
+  global:
+    secure: Xk+uGuB1bZGhSKVB+pfTrZaJ8JU0mlceIRbA2yLuHYsOpI3vpAY1YtNVaeZ9e3cUCc0uofvJHOnbdZiZFwEod9An2T6Iq939xFIAkHz9GyWJL54esldXFJaQscwtC9XO8o9V4eS7RbptTPRQpo3isPSk7+4bXLevCQ1lcE3xBoVjAQuGtoPRGODo0jfPiURN1QnKdfjKk0KIEb01YS5azdMq4WD8EV8IlF5eYb+3JUvzc2Le7hpPNzzmxco+VTLFUXIckJrKr4uJxDwj6v5bVaDq/JpzDsX4TKezSWVzZuLMDG/N21fwmA4U7iZVjHgHD279fU69lxINPvKKkfFEs4KXOPJ31W2S0eR2xbEq48kU9v00ixaQpqbPjBnhddOJBBOVpw0VF0BiW6n8drKMRqrz2UuWQVekPKJ609APhUXNZiF9gV5WWNElKuq0xIXpnQ3c2yww836w9Cj522hKQd6IGcqR/SWYvFgEHkLw17dJl6eSCT7N4slAoqLetr3JX/YTVwLMGvvYls0YANJPkaLr2Mv3NCiz5SIUz1zfK6nQEwx6X3Zr+HPw9enj90q0QV56o0MdgUN573HGivRO8b0Hb5hiq6gq1HkRZ+I/SUlMUZ64OaKMMIoYxlDvFicrFSnSbsK2PBWiFKF2A/ZX2QDyk9Y+SWIqa2K3ZWuGb0Q=
```
Comment (msftristew) [inline .travis.yml]: Explain?

Comment (aggFTW) [inline .travis.yml]: What part?

Comment (msftristew) [inline .travis.yml]: The long alphanumeric string :)

Comment (aggFTW) [inline .travis.yml]: https://docs.travis-ci.com/user/environment-variables/#Encrypted-Variables

**Comment by msftristew** [comment]
👍

File: autovizwidget/setup.py [hunk 2]
```diff
@@ -59,7 +59,7 @@ def version(path):
       install_requires=[
           'plotly>=1.10.0,<1.11.0',
           'ipywidgets>5.0.0',
-          'hdijupyterutils==0.2.3',
+          'hdijupyterutils>=0.3',
```
Comment (tc0312) [inline autovizwidget/setup.py]: We might want to add < 0.4 in the future to make sure it doesn't automatically upgrade to the next incompatible major release.

**Comment by tc0312** [comment]
LGTM besides the major version boundary in requirements.

**Comment by aggFTW** [comment]
Let's address that when we bump major version

**Comment by msftristew** [comment]
Before 1.0, every minor version bump is a potentially backwards-incompatible change per semver rules

**Comment by tc0312** [comment]
Cool.. Ship It.

**Pull request merged**